src/Illuminate/Database/Eloquent/Concerns/GuardsAttributes.php

```diff
@@ -41,7 +42,7 @@ trait GuardsAttributes
      */
     public function getFillable()
     {
-        return $this->fillable;
+        return !empty($this->fillable) ? $this->fillable : Schema::getColumnListing($this->getTable());
```

Comment:
Bad idea to make database queries as a side effect of calling this!

Reply:
How come?